dallinger/prolific.py

```diff
@@ -350,6 +350,27 @@ def publish_study(self, study_id: str) -> dict:
             json={"action": "PUBLISH"},
         )
 
+    def screen_out(
+        self,
+        study_id: str,
+        submission_id: str,
+        bonus_per_submission: float,
+        increase_places: bool,
+    ) -> dict:
+        """Screen-out a submission for a study."""
+
+        payload = {
```

Comment:
You should document what the expected return value of the function is, and explain what happens when the request is denied.

Reply:
Added extensive documentation incl. the one provide on the Prolific API docs page, see https://docs.prolific.com/docs/api-docs/public/#tag/Submissions/operation/BulkScreenOutSubmissions